changelog.d/7791.docker

```diff
@@ -0,0 +1 @@
+Add libwebp so we can handle webp images.
```

Comment:
I think this is a bit more accurate:

```suggestion
Include libwebp in the Docker file to properly handle webp image uploads.
```

Reply:
Ah, yes indeed. @juhovan feel free to credit yourself in the changelog entry as well (by adding "Contributed by..." to it) if you want!

Reply:
Thanks guys, that's better indeed :)